pom.xml

```diff
@@ -1291,6 +1299,17 @@
                                     <generator>java</generator>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>generate-thrift-sources-nodejs</id>
```

Comment:
Any reason for adding it?

Reply:
Admittedly this branch was never meant for being merged ... I used it to test my externalized thrift binaries and as I updated them to also contain the JS compiler, this was more added as a test for that.